apps/site/lib/site_web/views/partial_view.ex

```diff
@@ -78,102 +79,40 @@ defmodule SiteWeb.PartialView do
   def display_branch_name(_), do: nil
 
   @doc """
-  Renders a CMS content teaser, typically shown on a page's sidebar.
-  Guides only show their teaser image; other content types display
-  the content's topic and title below the teaser image.
+  Abstraction layer for displaying a list of teasers outside CMS context.
```

Comment:
I've sort of inferred what "outside CMS context" must mean here from our recent conversation, but I'm not sure if it would be obvious to others. Can we explain it? (e.g. "Renders CMS teasers outside the context of a CMS page, for example in the sidebar of an app page")